src/modes/OptimInterface.jl

```diff
@@ -50,14 +50,23 @@ function Base.show(io::IO, m::ModeResult)
     show(io, m.values.array)
 end
 
-function StatsBase.coeftable(m::ModeResult)
+function StatsBase.coeftable(m::ModeResult; level::Real=0.95)
     # Get columns for coeftable.
-    terms = StatsBase.coefnames(m)
-    estimates = m.values.array[:,1]
+    terms = String.(StatsBase.coefnames(m))
+    estimates = m.values.array[:, 1]
     stderrors = StatsBase.stderror(m)
-    tstats = estimates ./ stderrors
-
-    StatsBase.CoefTable([estimates, stderrors, tstats], ["estimate", "stderror", "tstat"], terms)
+    zscore = estimates ./ stderrors
+    p = 2 * (1 .- cdf.(Normal(0, 1), abs.(zscore)))
```

Comment:
This could be simplified to
```suggestion
    p = pvalue(Normal(), zscore; tail=:both)
```
(requires Distributions 0.25.90)